reference/forms/types/textarea.rst

```diff
@@ -28,6 +28,13 @@ Renders a ``textarea`` HTML element.
 | Class       | :class:`Symfony\\Component\\Form\\Extension\\Core\\Type\\TextareaType` |
 +-------------+------------------------------------------------------------------------+
 
+.. tip::
+
+    If you prefer to use an **advanced WYSIWYG editor** instead of a plain
+    textarea, consider using the `IvoryCKEditorBundle`_ community bundle. Read
```

Comment:
we can remove this reference, it does not exists and if anything, it would link to the same page as the "its documentation" link directly after it.